MirrorProvider/MirrorProvider.Linux/LinuxFileSystemVirtualizer.cs

```diff
@@ -3,6 +3,7 @@
 using System.IO;
 using System.Runtime.InteropServices;
 using System.Text;
+using Mono.Unix.Native;
```

Comment:
@wilbaker Can you comment on how we're doing platform native calls? Should we investigate if we should be using this library in the mac platform?

Reply:
fwiw, I tried a simple `[DllImport("libc")]` to get at the `stat` family of functions, but it didn't go down very well. I guess this is why `Mono.Posix.NETStandard` exists!